cargo-screeps/resources/default_initialization_header.js

```diff
@@ -2,6 +2,12 @@
 let wasm_module = null;
 
 function wasm_initialize() {
+    
+    if(Game.cpu.bucket < 500){
```

Comment:
We could have some spacing around the parens here, but that can always be changed afterwards.

Reply:
you can also edit the pr directly from the web interface if you want to fix style issues :)